mobile/src/bundles/auth/screens/auth.tsx

```diff
@@ -14,23 +14,22 @@ import {
     useCallback,
     useEffect,
 } from '~/bundles/common/hooks/hooks';
-import { actions as userActions } from '~/bundles/users/store';
 
 import { AuthWrapper, SignInForm, SignUpForm } from '../components/components';
+import { loadUser } from '../store/actions';
 
 const Auth: React.FC = () => {
     const { name } = useAppRoute();
     const dispatch = useAppDispatch();
     const { dataStatus } = useAppSelector(({ auth }) => ({
         dataStatus: auth.dataStatus,
     }));
-    const isSignUpScreen = name === AuthScreenName.SIGN_UP;
+
     const isPendingAuth = dataStatus === DataStatus.PENDING;
+
     useEffect(() => {
-        if (isSignUpScreen) {
-            void dispatch(userActions.loadAll());
-        }
-    }, [isSignUpScreen, dispatch]);
+        void dispatch(loadUser());
```

Comment:
It should be dispatched in root, user shouldn't see sign in screen each time he/she opens the app